### ISSUE TITLE
Multiline version_locations

### ISSUE THREAD

**Issue by pristupa** [sqlalchemy/alembic#1509]
**Describe the use case**
When there are many version locations it becomes difficult to read the value of the version_locations option. A new option "newline" for version_path_separator is suggested.

**Example Use**
```
version_locations =
  /foo/versions
  /bar/versions
  /baz/versions

version_path_separator = newline
```

**Have a nice day!**

**Comment by CaselIT** [comment]
hi,

seems like a reasonable request. Will look at the pr as soon as I've time

**Comment by CaselIT** [comment]
Closed by  #1510
